src/types/sdk.ts

```diff
@@ -31,6 +31,7 @@ export type SdkResponse = {
   document_back?: DocumentResponse
   document_video?: DocumentVideoResponse
   face?: FaceResponse
+  poa?: DocumentResponse
```

Comment:
```suggestion
  proof_of_address?: DocumentResponse
```

Reply:
However, maybe it makes more sense to keep `poa` as it is the capture name that exists already

Reply:
I'd be happy to make the change, or leave it the same. Just need to know which is truly preferred. 🙂

Reply:
Thanks, Sancha. As discussed elsewhere, we'll keep it as `poa`.